### PR TITLE
Add reusable stat component

### PR DESCRIPTION
**Pull request by priyesh777**
## Changes:
- Add reusable component such as status card which is used both in overview and country section

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

### CODE REVIEW

File: app/components/ProgressBar/index.tsx [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        ),
+        [
+            data,
+        ],
+    );
+
+    const tooltip = useMemo(
+        () => (
+            data.map((datum) => `${datum.title}: ${datum.value ?? 0}`).join('\n')
+        ), [data],
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx]:
```suggestion
        ),
        [data],
```

File: app/components/ProgressBar/index.tsx [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    );
+
+    return (
+        <>
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx]: Unncessary nesting

File: app/components/ProgressBar/index.tsx [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+                            <div
+                                key={item.title}
+                                className={styles.data}
+                                style={{
+                                    width: `${item.percentage}%`,
+                                    backgroundColor: item.color ?? 'blue',
+                                }}
+                            />
+                        );
+                    })}
```
Comment (AdityaKhatri) [inline app/components/ProgressBar/index.tsx]: Use list

File: app/views/Dashboard/Overview/MapView/index.tsx [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+                >
+                    {progressDataOne.map((item) => (
+                        <>
+                            {item.countryName}
+                            <ProgressBar
+                                barHeight={barHeight}
+                                data={item.progressData}
+                            />
+                        </>
+                    ))}
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/MapView/index.tsx]: Use listview

File: app/views/Dashboard/Overview/MapView/index.tsx [hunk 5]
```diff
@@ -0,0 +1,200 @@
...
+                >
+                    {progressDataTwo.map((item) => (
+                        <>
+                            {item.countryName}
+                            <ProgressBar
+                                barHeight={barHeight}
+                                data={item.progressData}
+                            />
+                        </>
+                    ))}
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/MapView/index.tsx]: Use listview

File: app/views/Dashboard/Overview/TableView/index.tsx [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+}
+
+function TableView(props: TableViewProps) {
+    const {
+        className,
+    } = props;
+
+    return (
+        <div className={_cs(className, styles.tableWrapper)}>
+            <ContainerCard>
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/TableView/index.tsx]: use only ContainerCard

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by shreeyash07 — APPROVED** [review]

**Review by shreeyash07 — APPROVED** [review]

**Review by shreeyash07 — APPROVED** [review]

File: app/views/Dashboard/Overview/MapView/index.tsx [hunk 7]
```diff
@@ -0,0 +1,215 @@
...
+                    mapStyle={lightStyle}
+                    mapOptions={{
+                        logoPosition: 'bottom-left',
+                    }}
+                    scaleControlShown
+                    navControlShown
+                >
+                    <MapContainer className={styles.mapContainer} />
+                    <MapBounds
+                        bounds={[80.088425, 26.397898, 88.174804, 30.422717]}
```
Comment (AdityaKhatri) [inline app/views/Dashboard/Overview/MapView/index.tsx]: define this outside

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**